coffeecard/CoffeeCard.WebApi/Startup.cs

```diff
@@ -76,6 +76,14 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<Library.Services.v2.IAccountService, Library.Services.v2.AccountService>();
             services.AddScoped<IPurchaseService, PurchaseService>();
             services.AddScoped<IMapperService, MapperService>();
+            if (_environment.IsDevelopment())
```

Comment:
I am realizing that this should be LocalDevelopment as we only want the SMTP client on local, not the remote dev environment

Reply:
Hmm, since we don't override the ASPNETCORE_ENVIRONMENT variable in our deployment, I would expect it to be set to production per default since it is a published application. If you want to use one that we do set, use the one from the EnvironmentSettings.